### DISCUSSION TITLE
Cloud Security Alliance

### DISCUSSION THREAD

**Discussion by AdrianHammond** [finos/common-cloud-controls#193] [General]
I gave an overview of Common Cloud Controls at the OpenUK Security meetup. I was asked if Cloud Security Alliance [Cloud Controls Matrix and Consensus Assessment Initiative Questionnaire](https://cloudsecurityalliance.org/research/cloud-controls-matrix) had been looked at as it was felt that they might be of use to CCC.  @mlysaght2017 any views from security WG?  The person who asked me offered to discuss if needed.  Thanks Adrian

**Comment by mlysaght2017** [comment]
Thanks @AdrianHammond  - would be very interested to discuss more with csa contact if you can help set something up.

**Comment by iMichaela** [comment]
@mlysaght2017 and @AdrianHammond - I've been collaborating closely with CSA for the past 14 years.  CSA's CCM is expressed in OSCAL as well and it is mapped (crosswalks to many other security standards.  I can introduce Daniele Catteddu , CTO , CSA to anyone interested in this group.  You can find his talk and panel recorded [here](https://www.nist.gov/news-events/events/2023/05/4th-open-security-controls-assessment-language-oscal-conference-and) during the 4th Annual OSCAL Conference. 
PS. Other presenters contributing to the CCC effort are Google (you know Val Mihai), and Dr Anca Sailer (IBM Trestle Platform lead)
I did not initiate any invitation or suggested any since this WG is a membership based or at least this is my understanding ...

**Comment by mlysaght2017** [comment]
Thanks @iMichaela - I'd be very grateful for an introduction.

**Reply by iMichaela** [reply]
Do you mind pinging me at michaela@nist.gov so I have your direct email address? Thank you.

**Comment by damienjburks** [comment]
@AdrianHammond - are we good to close this out?

**Comment by AdrianHammond** [comment]
Hi @damienjburks - I am happy this can be closed out.

**Comment by AdrianHammond** [comment]
closing
